sdk/mediaservices/arm-mediaservices/README.md

```diff
@@ -24,25 +24,40 @@ npm install @azure/arm-mediaservices
 npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
-##### Sample code
+##### Sample code using Service Principal to list all Assets
 
 ```typescript
 import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
 import { AzureMediaServices, AzureMediaServicesModels, AzureMediaServicesMappers } from "@azure/arm-mediaservices";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new AzureMediaServices(creds, subscriptionId);
-  const resourceGroupName = "testresourceGroupName";
-  const accountName = "testaccountName";
-  client.accountFilters.list(resourceGroupName, accountName).then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
-}).catch((err) => {
-  console.error(err);
+
+export async function main() {
+    // Go to the Azure Portal and copy the values obtained 
+    // from your Media Services account's API Access page into the constants
+    const clientId = "<<Enter the AadClientId value from the Azure Portal>>";
+    const secret = "<<Enter the AadSecret value from the Azure Portal>>";
+    const tenantDomain = "<<Enter the AadTenantDomain value from the Azure portal>>";
+    const subscriptionId = "<<Enter the SubscriptionId value from the Azure portal>>";
+    const resourceGroup = "<<Enter the ResourceGroup value from the Azure portal>>";
+    const accountName = "<<Enter the AccountName value from the Azure portal>>";
+
+
+    const creds = await msRestNodeAuth.loginWithServicePrincipalSecret(clientId, secret, tenantDomain);
+    const mediaClient = new AzureMediaServices(creds, subscriptionId);
+
+    // List Assets in Account
+    console.log("Listing Assets Names in account:")
+    var assets = await mediaClient.assets.list(resourceGroup, accountName);
+
+    assets.forEach(asset => {
+        console.log(asset.name);    
+    });
+
+}
+
+main().catch((err) => {
+    console.error("Error running sample:", err.message);
 });
```

Comment:
This section is followed by a browser sample. The current auth flow is fine, but you want to change the listing of accountfilters to listing of assets there as well

Reply:
I'll change the browser sample later once i test that out.

Reply:
One more change - take a look now @ramya-rao-a